src/module_EARTH_GRID_COMP.F90

```diff
@@ -3596,8 +3599,21 @@ subroutine SetModelServices(driver, rc)
               file=__FILE__, rcToReturn=rc)
             return  ! bail out
 #endif
-          elseif (trim(model) == "datm") then
-#ifdef FRONT_DATM
+          elseif (trim(model) == "nems_datm") then
+#ifdef FRONT_NEMS_DATM
```

Comment:
@binli2337 @junwang-noaa I think we don't need to have same code twice in here. You could use `#if defined FRONT_NEMS_DATM || defined FRONT_CDEPS_DATM` and remove extra if statement in here. This is already working under HAFS fork.